instructor-notes/pr-review.md

```diff
@@ -20,8 +20,10 @@ Before the live demo, the co-instructor (i.e., instructor who is _not_ leading t
     * Need to set appropriate `ggplot` theme for the UMAP (no axis ticks, labels)
   * File-level comment
     * Need to include more markdown text contextualizing code chunks
-* Check out branch locally to run the code, during which a bug is caught because `{umap}` is not loaded into environment
+* Check out branch locally to run the code, during which package bugs are caught:
+  * `{umap}` was not loaded into environment
+  * `{patchwork}` was used, but the reviewer does not have the package installed
```

Comment:
I'm being pedantic, but this is not a bug like the first point – it's a matter of something not being managed yet.